### PR TITLE
adding setTimeout term

### PR DESCRIPTION
**Pull request by RaffyRod**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by RaffyRod** [comment]
@yangc95  @KTom101 Sorry, this is the new MR with the changes suggested before, and ready for review.

File: content/javascript/concepts/setTimeout/setTimeout.md
Comment (SSwiniarski) [inline content/javascript/concepts/setTimeout/setTimeout.md:54]:
```suggestion
The following example shows the use of [arrow functions](https://www.codecademy.com/resources/docs/javascript/arrow-functions). Instead of declaring it using the word `function`, we can just create it by using `() =>`.
```
Such references work better in-line.

File: content/javascript/concepts/setTimeout/setTimeout.md
Comment (SSwiniarski) [inline content/javascript/concepts/setTimeout/setTimeout.md:59]:
```suggestion
  console.log('Hello World from Codecademy');
```

File: content/javascript/concepts/setTimeout/setTimeout.md
Comment (SSwiniarski) [inline content/javascript/concepts/setTimeout/setTimeout.md:66]:
```suggestion
Hello World from Codecademy
```

File: content/javascript/concepts/setTimeout/setTimeout.md
Comment (SSwiniarski) [inline content/javascript/concepts/setTimeout/setTimeout.md:70]: Have one sentence introducing the codebyte explaining what it does.

File: content/javascript/concepts/setTimeout/setTimeout.md
Comment (SSwiniarski) [inline content/javascript/concepts/setTimeout/setTimeout.md:20]:
```suggestion
`setTimeout()` is capable of receiving multiple parameters where the first is a callback function, the second parameter receives a number that represents the time in milliseconds (1s = 1000ms), that defines the time needed for the callback to execute, the third parameter onwards will be the parameters that the callback function would take in case arguments are defined within the callback.
```
We only bold the first occurrence of the term.

File: content/javascript/concepts/setTimeout/setTimeout.md
Comment (SSwiniarski) [inline content/javascript/concepts/setTimeout/setTimeout.md:3]:
```suggestion
Description: 'Allows users to execute callbacks after a period of time expressed in milliseconds.'
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@RaffyRod I did a first review and left a few comments. Overall a strong entry, I only suggested a few line edits here and there.

**Comment by RaffyRod** [comment]
@SSwiniarski  requested changes were added.

File: content/javascript/concepts/setTimeout/setTimeout.md
Comment (SSwiniarski) [inline content/javascript/concepts/setTimeout/setTimeout.md:16]:
```suggestion
**`setTimeout()`** is a function serviced globally by the window object provided by the user's browser. It allows users to execute callbacks after a period of time expressed in milliseconds.
```
Information in the description should be in the first paragraph of the entry.

File: content/javascript/concepts/setTimeout/setTimeout.md
Comment (SSwiniarski) [inline content/javascript/concepts/setTimeout/setTimeout.md:22]:
```suggestion
Below are some examples of how `setTimeout()` is composed:
```

File: content/javascript/concepts/setTimeout/setTimeout.md
Comment (SSwiniarski) [inline content/javascript/concepts/setTimeout/setTimeout.md:34]:
```suggestion
The following example shows an example of the `setTimeout()` implementation:
```

File: content/javascript/concepts/setTimeout/setTimeout.md
Comment (SSwiniarski) [inline content/javascript/concepts/setTimeout/setTimeout.md:71]:
```suggestion
```
You misunderstand. We don't need an explanation of what a codebyte is, we want an explanation of what the code in it does.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@RaffyRod I went through and left a few more comments.

**Review by SSwiniarski — APPROVED** [review]
@RaffyRod Looks good for a second review.

File: content/javascript/concepts/setTimeout/setTimeout.md
Comment (KTom101) [inline content/javascript/concepts/setTimeout/setTimeout.md:54]: Avoid the use of pronouns like we, you, and us.
```suggestion
The following example uses [arrow functions](https://www.codecademy.com/resources/docs/javascript/arrow-functions). Instead of declaring the function using the keyword `function`, it is created using `() =>`.
```

File: content/javascript/concepts/setTimeout/setTimeout.md
Comment (KTom101) [inline content/javascript/concepts/setTimeout/setTimeout.md:34]:
```suggestion
The following example uses the `setTimeout()` function to set a timer to execute the `sayHello()` function:
```

File: content/javascript/concepts/setTimeout/setTimeout.md
Comment (KTom101) [inline content/javascript/concepts/setTimeout/setTimeout.md:20]:
```suggestion
`setTimeout()` is capable of receiving multiple parameters where the first is a callback function. The second parameter receives a number that represents the time in milliseconds (1s = 1000ms), which defines the time needed for the callback to execute. The third parameter onwards will be the parameters that the callback function would take in case arguments are defined within the callback.
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@RaffyRod I've added some more changes

**Comment by RaffyRod** [comment]
@KTom101 please check now

**Review by KTom101 — APPROVED** [review]
@RaffyRod LGTM 👍 Currently we're waiting for the JavaScript Window concept to be up, which [the issue](https://github.com/Codecademy/docs/issues/2007) has been claimed by @SSwiniarski . Merging this as-is will not render with the Concept entry missing. So chill out for now and keep this PR open. If anything needs to be added during the wait, we can also do that. Additionally, don't worry about updating the branch, we will do this when we do the final merge. 

I will also tag @yangc95 in this just in case they see anything amiss from [the original PR](https://github.com/Codecademy/docs/pull/2114).

**Comment by RaffyRod** [comment]
@KTom101 perfect, I'll wait

File: content/javascript/concepts/setTimeout/setTimeout.md
Comment (RaffyRod) [reply content/javascript/concepts/setTimeout/setTimeout.md:1]: done

**Comment by SSwiniarski** [comment]
@RaffyRod Thanks for your patience waiting for this to merge. And congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/javascript/window/setTimeout

image

**Pull request merged**
